catalogue/webapp/components/ImageSearchResults/ImageSearchResults.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import { useState } from 'react';
+import ExpandedImage from '../ExpandedImage/ExpandedImage';
+import ImageCard from '../ImageCard/ImageCard';
+import { trackSearchResultSelected } from '@weco/common/views/components/Tracker/Tracker';
+import { type CatalogueResultsList } from '@weco/common/model/catalogue';
+import { type CatalogueApiProps } from '@weco/common/services/catalogue/api';
+
+type Props = {|
+  works: CatalogueResultsList,
+  apiProps: CatalogueApiProps,
+|};
+
+const ImageSearchResults = ({ works, apiProps }: Props) => {
+  const [expandedImageId, setExpandedImageId] = useState('');
+
+  return (
+    <div className={'flex flex--wrap'}>
+      {works.results.map((result, i) => (
+        <div key={result.id}>
+          <div
+            onClick={() => {
+              trackSearchResultSelected(apiProps, {
+                id: result.id,
+                position: i,
+                resultWorkType: result.workType.label,
+                resultLanguage: result.language && result.language.label,
+                resultIdentifiers: result.identifiers.map(
+                  identifier => identifier.value
+                ),
+                resultSubjects: result.subjects.map(subject => subject.label),
+              });
+            }}
+          >
+            <ImageCard
+              id={result.id}
+              image={{
+                contentUrl: result.thumbnail
+                  ? result.thumbnail.url
+                  : 'https://via.placeholder.com/1600x900?text=%20',
```

Comment:
Moving over some nasty bits too?

Reply:
😳 Yeah can't pretend I didn't see this when I copy/pasted it over. Do we know if this is even necessary – i.e. are there any results that don't have a thumbnail anymore?

Reply:
In the catalogue - yes.
Not from the set of results we're returning.

So by typing we should cater for this. Could we leave a message - or, as we're doing, a grey box. Was just making sure we thought about it.